canvas_modules/common-canvas/src/common-canvas/svg-canvas-renderer.js

```diff
@@ -4353,7 +4355,9 @@ export default class SVGCanvasRenderer {
 					CanvasUtils.stopPropagationAndPreventDefault(d3Event);
 
 					this.deleteCommentPort(d3Event.currentTarget);
-					this.displayCommentTextArea(d, d3Event.currentTarget);
+					// this.displayCommentTextArea(d, d3Event.currentTarget);
```

Comment:
Is this comment left in intentionally?

Reply:
Oops no it wasn't. Sorry about that -- I have removed it.